get-shit-done/workflows/new-project.md

```
@@ -410,25 +410,80 @@ gsd-sdk query commit "docs: initialize project" .planning/PROJECT.md
+- `mode` → "Mode"
+- `granularity` → "Granularity"
+- `parallelization` → "Execution" (`true` → "Parallel", `false` → "Sequential")
+- `commit_docs` → "Git Tracking" (`true` → "Yes", `false` → "No")
+- `model_profile` → "AI Models"
+- `workflow.research` → "Research agent" (`true` → "Yes", `false` → "No")
+- `workflow.plan_check` → "Plan Check agent" (`true` → "Yes", `false` → "No")
+- `workflow.verifier` → "Verifier agent" (`true` → "Yes", `false` → "No")
+
+Display above the prompt:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)